dev/tasks/tasks.yml

```diff
@@ -856,6 +857,16 @@ tasks:
       - flight-grpc-{no_rc_version}-tests.jar
       - flight-grpc-{no_rc_version}.jar
       - flight-grpc-{no_rc_version}.pom
+      - flight-integration-tests-{no_rc_version}-javadoc.jar
```

Comment:
double-checking these and looks like I missed the jar-with-dependencies. I'll try again, third times the charm!